### PR TITLE
Update loading of packages to support being used on Drupal8/9 which i…

### PR DESCRIPTION
**Pull request by seamuslee001**
…s needed for the contact layout editor

ping @ginkgomzd @colemanw 

I also took the opportunity to bump up the release and min version to be the version when packagesBase  was defined as per https://github.com/civicrm/civicrm-core/commit/3130950c51d3ceadc913495e8dc52125bb22503e

### CODE REVIEW

File: info.xml [hunk 1]
```diff
@@ -8,12 +8,11 @@
     <author>Ginkgo Street Labs</author>
     <email>inquire@ginkgostreet.com</email>
   </maintainer>
-  <releaseDate>2018-06-29</releaseDate>
-  <version>4.7.31-1.1.2</version>
+  <releaseDate>2020-10-13</releaseDate>
+  <version>4.7.31-1.2.0</version>
```
Comment (colemanw) [inline info.xml:12]: The versioning convention in this extension is unique, but if we follow the pattern then this version number should be `5.23-1.2.0`

**Comment by caracolmzd** [comment]
@colemanw I pushed a new tag... am I reading the docs correctly that to publish to the extension directory, it is now required it be hosted on GitLab?

**Comment by colemanw** [comment]
No I don't think GL is required. Just needs to be a public `git` repository.

**Comment by colemanw** [comment]
@ginkgomzd the only issue with your tag is that it doesn't match the version number in the xml. In order for publishing to work, they have to match.

**Comment by caracolmzd** [comment]
It looks like it just needed time to process. I did update info.xml after I merged this PR.

In the docs, it set my expectation there was a button I should click in the extension directory.
https://docs.civicrm.org/dev/en/latest/extensions/publish/#publishing-an-extension-release

Maybe the button only appears before the first release?

I should have read the last bullet that implies I should wait a day for processing:
"Within a day, you will receive an email notifying you that the release was published on civicrm.org or that a problem with the extension manifest prevented publication."

**Pull request merged**